docs/platform/reference/end-of-life.md

```diff
@@ -0,0 +1,58 @@
+---
+title: End of life for Aiven services
+---
+
+Learn about the upcoming end of life (EOL) for select Aiven services, including timelines, actions after end of life, recommended migration options, and next steps.
+
+## Aiven for M3DB
+
+**Timeline**: End of life (EOL) on April 30, 2025
```

Comment:
I'm not sure a timeline can be define with one date only? Maybe a simple `Date` is more accurate?

Reply:
updated to date.